modules/express/src/clientRoutes.ts

```diff
@@ -1449,6 +1554,21 @@ export function setupAPIRoutes(app: express.Application, config: Config): void {
   app.use('/api/v2/user/*', parseBody, prepareBitGo(config), promiseWrapper(handleV2UserREST));
   app.use('/api/v2/:coin/*', parseBody, prepareBitGo(config), promiseWrapper(handleV2CoinSpecificREST));
 
+  app.post(
+    '/api/network/v1/enterprises/:enterpriseId/clients/connections',
+    parseBody,
+    prepareBitGo(config),
+    promiseWrapper(handleNetworkV1EnterpriseClientConnections)
+  );
+
+  app.use(
+    '/api/:namespace/v[12]/enterprises/:enterpriseId/*',
+    parseBody,
+    prepareBitGo(config),
+    promiseWrapper(handleMicroserviceReq)
+  );
+  app.use('/api/:namespace/v[12]/*', parseBody, prepareBitGo(config), promiseWrapper(handleMicroserviceReq));
+
```

Comment:
Adding these `.use`s allows for these calls to be properly handled for namespaced apis, not only the /api/v[12] that were only being handled previously.

Reply:
https://github.com/BitGo/BitGoJS/assets/100626066/2655f27b-9eca-494a-906e-e1022ded4502

Reply:
https://github.com/BitGo/BitGoJS/assets/100626066/b5a72e0a-588d-41e3-a6a9-30bc030f2a4a